### DISCUSSION TITLE
GitHub Actions is currently disabled for your account<title>

### DISCUSSION THREAD

**Discussion by Djemchik** [community/community#47582] [Actions]
### Select Topic Area

General

### Body

I have a problem with GitHub Actions "GitHub Actions is currently disabled for your account"

**Comment by mrvcoder** [comment]
Hello , me too 
Why?

**Comment by Akash1134** [comment]
Hi @Djemchik and @mrvcoder, if you are having issues with GitHub Actions being disabled on your account, please submit a [GitHub Support ticket](https://support.github.com/request). They will perform a check/investigation on why that setting was disabled and get back to you.

**Reply by shubhankartrivedi** [reply]
Yeah that's all what we did. But imagine waiting for a year to get a reply on support ticket and still don't get any reply......

Its like the whole GitHub support is dead right now....
Was about to buy GitHub pro but this feels very bad....
